packages/embed/README.md

```diff
@@ -126,6 +126,7 @@ Closing and opening a typeform in modal window will restart the progress from th
 | [height](https://codesandbox.io/s/github/Typeform/embed-demo/tree/main/demo-html/popup-html)                                 | number / string             | height of the embed - number in pixels or string including units, supported by all embeds except slider (for popup you can specify `size` instead)                                                                                                                                                     | `undefined`                                                   |
 | [hidden](https://codesandbox.io/s/github/Typeform/embed-demo/tree/main/demo-html/widget-js)                                  | object                      | [hidden fields](https://help.typeform.com/hc/en-us/articles/360050448072-Hidden-fields-explained) to be passed to the form in [URL hash](https://developer.mozilla.org/en-US/docs/Web/API/URL/hash)                                                                                                    | `undefined`                                                   |
 | [tracking](https://codesandbox.io/s/github/Typeform/embed-demo/tree/main/demo-html/widget-html)                              | object                      | [tracking parameters](https://help.typeform.com/hc/en-us/articles/360050914311-What-is-UTM-tracking-) to be passed to the form in [URL query string](https://developer.mozilla.org/en-US/docs/Web/API/URL/search)                                                                                      | `undefined`                                                   |
+| hubspot                                                                                                                      | boolean                     | enable [HubSpot tracking](https://www.typeform.com/help/a/hub-spot-integration-installation-and-setup-360029578071/)                                                                                                                                                                                   | `false`                                                       |
```

Comment:
This doc already has `hubspot` in the table.

Reply:
Oh, sorry I didn't see it, I removed it.